_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+
+_After being a teacher for a fair share of years, I wanted to work with the industry on new emerging technologies and challenge myself. Besides that, I always wanted to work on an open-source project. It is really exciting to be learning and building with such great devs and user community._
+
+#### What do you like to do in your free time?
+
+_I enjoy playing digital games such as ~~cracktorio~~ Factorio and StarCraft II, but also like to gather some friends and play board games while drinking some beers 🍻. Other than that, I really like skateboarding, running and football of course 🇧🇷!_
+
+
+### Nikola - @[nikola-matic](https://github.com/nikola-matic)
+
+#### What is your professional background?
+
+_I have a BSc in Comp Sci, and a ~10 year long obsession with C++, which kinda narrowed down my options as far as jobs were concerned, because no one's really gonna write a web app in C++. Having said that, I ended up working on a ton of interesting projects throughout my career, like writings high-perf server-client applications for SMS messaging, hardware virtualization on the cloud, EV routing, and a ton of other stuff. Unfortunately, up until I joined the EF and the Solidity team, I've never worked on an open source project, so this is a first for me (hopefully not the last)._
+
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+_I've only joined the team ~4 months prior to writing this, so not that much. I did implement the option for the user to pass the clean-up steps for the optimizer sequence (both CLI and JSON), so at least that's something. At the moment, and for the foreseeable future, I'll be working on helping Chris (chriseth) with speeding up the `--via-ir` pipeline, as well as getting the experimental version of the standard library merged._
+
+#### What do you think is Solidity’s biggest challenge?
+
+_Fundamentally, I think that we as the Solidity team don't use the language enough, which is especially concerning seeing as we design and implement it. This is something that the community has pointed out in the past as well, and is something we're well aware of. Thankfully, we are working on this, and plan to have internal team projects and hackathons in the future, which should hopefully give us a different perspective going forward._
+
+
+### Nuno - @[NunoFilipeSantos](https://github.com/NunoFilipeSantos)
+
+#### What is your professional background?
+
+_I have a MSc in Software Engineering and all my life I have been highly motivated to work with open-source software. In recent years, I have worked for SUSE, the largest independent open-source company in the world, and Timescale, an open-source time-series database extension for PostgreSQL as a Product Manager. I am usually also an informal Agile coach for most of my teams with a heavy focus on execution and team happiness/motivation. I have been a remote worker for almost 10 years now and have been 100% remote since 2018._
```

Comment:
```suggestion
_I have an MSc in Software Engineering, and all my life I have been highly motivated to work with open-source software. In recent years, I have worked for SUSE, the largest independent open-source company in the world, and Timescale, an open-source time-series database extension for PostgreSQL as a Product Manager. I am usually also an informal Agile coach for most of my teams, with a heavy focus on execution and team happiness/motivation. I have been a remote worker for almost 10 years now and have been 100% remote since 2018._
```